### PR TITLE
Update domains.json 

### PR DESCRIPTION
**Pull request by Dog-Owner**
Add a custom domain - gov.dogownertoken.com

- [x ] Add a custom domain for your space

### CODE REVIEW

File: spaces/domains.json [hunk 1]
```diff
@@ -1,4 +1,6 @@
 {
+"gov.dogownertoken.com": "dogown.eth"
+}
```
Comment (gokaiorg) [inline spaces/domains.json:3]: Please remove the extra bracket and add a comma.
```suggestion
"gov.dogownertoken.com": "dogown.eth",
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Review by Dog-Owner — COMMENTED** [review]
Brackets removed and comma added

File: spaces/domains.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{
```
Comment (gokaiorg) [inline spaces/domains.json:1]: Please do not delete this bracket.

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Review by gokaiorg — APPROVED** [review]

**Comment by Dog-Owner** [comment]
Thank you @gokaiorg

**Pull request merged**